src/derivative_wrappers.jl

```diff
@@ -692,6 +692,8 @@ function _vecjacobian!(dλ, y, λ, p, t, S::TS, isautojacvec::EnzymeVJP, dgrad,
 
     isautojacvec = get_jacvec(sensealg)
 
+    Enzyme.make_zero!(_tmp6)
```

Comment:
above where you have the tmp2/3 .= 0, you may be able to replace that with make_zero! as well

Reply:
I had to revert those due to https://github.com/EnzymeAD/Enzyme.jl/issues/1611